sphinx/application.py

```diff
@@ -205,9 +205,9 @@ def __init__(self, srcdir, confdir, outdir, doctreedir, buildername,
                 self.config.setup(self)
             else:
                 raise ConfigError(
-                    _("'setup' that is specified in the conf.py has not been "
-                      "callable. Please provide a callable `setup` function "
-                      "in order to behave as a sphinx extension conf.py itself.")
+                    _("'setup' that is defined in the conf.py, is not a Python callable. "
+                      "Please provide a callable `setup` function in order to make conf.py "
+                      "behave as a Sphinx extension.")
```

Comment:
oops there was a typo in my proposal

    _("'setup'" as currently defined in conf.py isn't a Python callable. "
      "Please modify its definition to make it a callable function. This is "
      "needed for conf.py to behave as a Sphinx extension.")